arch/rv32i/src/pmp.rs

```diff
@@ -43,7 +43,9 @@ register_bitfields![u8,
 /// this mask achieves the same effect; thus it can be used to determine whether
 /// a given PMP region spec would be legal and applied before writing it to a
 /// `pmpaddrX` CSR.
-const PMPADDR_RV64_MASK: u64 = 0x003F_FFFF_FFFF_FFFF;
+/// This mask will have the value `0x003F_FFFF_FFFF_FFFF` on RV64 platforms, and
+/// `0xFFFFFFFF` on RV32 platforms.
+const PMPADDR_RV64_MASK: usize = (0x003F_FFFF_FFFF_FFFFu64 & usize::MAX as u64) as usize;
```

Comment:
This mask is now generic across both RV32 and RV64 platforms. Can you remove `RV64` from the name, and in the doc-comment above? Also, please add an empty new-line before your additions to the doc-comment, otherwise this will be not be displayed as a separate paragraph in the rendered Markdown.